Package.swift

```diff
@@ -0,0 +1,47 @@
+// swift-tools-version:5.3
```

Comment:
Do we actually need 5.3 for this? I think we could probably go back to 5.0.1 and be fine.

Reply:
The syntax required to install SnapshotTesting doesn't exist that far back. Earliest I could find was 5.2.
Also we require the `defaultLocalization` due to the strings files the project contains - this wasn't added until 5.3 (see [here](https://github.com/apple/swift-evolution/blob/master/proposals/0278-package-manager-localized-resources.md))

Reply:
Cool, leaving it at 5.3 sounds good then 👍🏻